src/test/kfs_rest_test.cpp

```diff
@@ -171,3 +199,35 @@ TEST_F(HttpRestApiHandlerTest, dispatchReady) {
 
     ASSERT_EQ(c, 1);
 }
+
+TEST_F(HttpRestApiHandlerTest, serverReady) {
+    ovms::HttpRequestComponents comp;
+    comp.type = ovms::KFS_GetServerReady;
+    std::string request;
+    std::string response;
+    ovms::Status status = handler->dispatchToProcessor(request, &response, comp);
+
+    ASSERT_EQ(status, server->isReady() ? ovms::StatusCode::OK : ovms::StatusCode::MODEL_NOT_LOADED);
+}
+
+TEST_F(HttpRestApiHandlerTest, serverLive) {
+    ovms::HttpRequestComponents comp;
+    comp.type = ovms::KFS_GetServerLive;
+    std::string request;
+    std::string response;
+    ovms::Status status = handler->dispatchToProcessor(request, &response, comp);
+
+    ASSERT_EQ(status, server->isLive() ? ovms::StatusCode::OK : ovms::StatusCode::INTERNAL_ERROR);
+}
```

Comment:
changing expected status here does not make sense since you have started server in this test suite and it will always respond with Live.